src/main/java/homeworks/juanpa/firstAssigment/composition/Engineer.java

```diff
@@ -0,0 +1,11 @@
+package homeworks.juanpa.firstAssigment.composition;
+
+public class Engineer {
```

Comment:
Always remember about JavaDoc, also there's is no extends therefore there is no inheritance.